### ISSUE TITLE
Small Note

### ISSUE THREAD

**Issue by kaledhoshme123** [AhmedNasser1601/Diabetic-Retinopathy-Detection#2]
Well done @AhmedNasser1601 
Great job but I have a small note:
There is an overfitting case between accuracy and loss curves for both the training data and the test data, in both proposed models when categorizing 5 or 2 classes.
**So I have some suggestions:**
- At first, reduce the learning rate to a small value and increase the number of epochs.
- Increase the number of neurons for the Dense layer, and simultaneously use the Dropout layer with a high percentage.
- Use tf.keras.callbacks.ReduceLROnPlateau to lower the learning rate values during the training process.

**Comment by AhmedNasser1601** [comment]
Hi @kaledhoshme123 ,
Thank you for your positive feedback and your fantastic notes
**It 'll be considered**
